### PR TITLE
 Counter component - Adding aria-live attribute on input element

### PR DESCRIPTION
**Pull request by barboaron**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6757/

File: packages/wix-ui-tpa/src/components/Counter/Counter.tsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ export interface CounterProps extends TPAComponentProps {
   incrementAriaLabel: string;
   decrementAriaLabel: string;
   inputAriaLabel: string;
+  inputAriaLive?: 'off' | 'assertive' | 'polite';
```
Comment (nitayneeman) [inline packages/wix-ui-tpa/src/components/Counter/Counter.tsx:22]: Maybe we can create an enum to reuse:
```tsx
export enum AriaLiveOptions {
  off = 'off',
  assertive = 'assertive',
  polite = 'polite',
}
```

Like this:
https://github.com/wix-private/wix-style-react/blob/master/packages/wix-ui-tpa/src/components/Image/types.ts#L35-L38


And then:
```suggestion
  inputAriaLive?: AriaLiveOptions
```

File: packages/wix-ui-tpa/src/components/Counter/Counter.tsx [hunk 2]
```diff
@@ -35,6 +36,7 @@ interface DefaultProps {
   step: number;
   value: number;
   size: CounterSize;
+  inputAriaLive: 'off' | 'assertive' | 'polite';
```
Comment (nitayneeman) [inline packages/wix-ui-tpa/src/components/Counter/Counter.tsx:39]: Also here we can use the type

File: packages/wix-ui-tpa/src/components/Counter/Counter.tsx [hunk 3]
```diff
@@ -44,6 +46,7 @@ export class Counter extends React.Component<CounterProps> {
     step: 1,
     value: 0,
     size: CounterSize.medium,
+    inputAriaLive: 'off',
```
Comment (nitayneeman) [inline packages/wix-ui-tpa/src/components/Counter/Counter.tsx:49]: And here as well:
```suggestion
    inputAriaLive: AriaLiveOptions.off,
```

**Review by nitayneeman — COMMENTED** [review]
Very nice!

I added little suggestions.

And also I think we should add the prop to the meta testing:
https://github.com/wix-private/wix-style-react/blob/master/packages/wix-ui-tpa/src/components/Counter/Counter.meta.tsx#L14

File: packages/wix-ui-tpa/src/components/Counter/Counter.tsx (hunk 1 above)
Comment (barboaron) [reply packages/wix-ui-tpa/src/components/Counter/Counter.tsx:22]: done

File: packages/wix-ui-tpa/src/components/Counter/Counter.tsx (hunk 2 above)
Comment (barboaron) [reply packages/wix-ui-tpa/src/components/Counter/Counter.tsx:39]: done

File: packages/wix-ui-tpa/src/components/Counter/Counter.tsx (hunk 3 above)
Comment (barboaron) [reply packages/wix-ui-tpa/src/components/Counter/Counter.tsx:49]: done

**Comment by nitayneeman** [comment]
@NeilWix - we wonder if `aria-live` should be a prop accepting `off`/`assertive`/`polite` or a hardcoded value that always be `assertive`? 

FYI @jonathanadler

**Comment by NeilWix** [comment]
In the case of the counter component, assertive should do. In other cases, it's indeed needed

**Review by nitayneeman — APPROVED** [review]

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6757/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6757

**Pull request merged**